Py time (#669)
Codecademy/docs: content/python/concepts/time-module/terms/time-ctime/time-ctime.md

---

> Title: 'time.ctime()'

Reviewer: ```suggestion
Title: '.ctime()'
```

Author: @Dusch4593 Do we have a reference document that includes these details. I modeled this entry on the py math module which uses the format I've applied here. I feel like I'm always referencing formatting that isn't considered the "current" standard.

> Title: 'time.ctime()'

Reviewer: ```suggestion
Title: '.ctime()'
```

> Description: 'Returns a time expressed in seconds (since the epoch) to a string format.'

Reviewer: ```suggestion
Description: 'Returns a formatted string of the time, expressed in second, since the epoch.
```
```suggestion
Description: 'Returns a time expressed in seconds (since the epoch) to a string format.'
```

> Description: 'Returns a time expressed in seconds (since the epoch) to a string format.'

Reviewer: ```suggestion
Description: 'Returns a formatted string of the time, expressed in second, since the epoch.'
```

Author: Confused by these edits- why is this line highlighted three times?

> Description: 'Returns a formatted string of the time elapsed since the epoch, expressed in seconds.'

Reviewer 2: ```suggestion
Description: 'Takes a time, expressed in seconds elapsed since the epoch, and returns a formatted string.'
```

Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Web Development'
Tags:
  - 'Methods'
  - 'Constants'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/front-end-engineer-career-path'
---

> Returns a time expressed in seconds (since the epoch) to a string format.

Reviewer: ```suggestion
The `ctime()` function returns a formatted string of the time, expressed in seconds, since the epoch.
```

> Returns a time expressed in seconds (since the epoch) to a string format.

Reviewer: ```suggestion
The `ctime()` function returns a formatted string of the time, expressed in seconds, since the epoch.
```

> The `ctime()` function returns a formatted string of the time elapsed, expressed in seconds, since the epoch.

Reviewer 2: ```suggestion
The `ctime()` function takes a time, expressed in seconds elapsed since the epoch, and returns a formatted string.
```

## Syntax

```pseudo
ctime(s)
```

> The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00. If no argument is provided the function will return the current local time.

Reviewer 2: ```suggestion
The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00. If no argument is provided the function will use the current local time.
```

> The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00. If no argument is provided the function will use the current local time.

Reviewer 3: ```suggestion
The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00 UTC. If no argument is provided the function will use the current local time.
```
